jaxb-ri/samples/src/main/samples/spec-samples/runtime/javabean-nosetter/Main.java

```diff
@@ -1,86 +0,0 @@
-/*
- * Copyright (c) 1997, 2018 Oracle and/or its affiliates. All rights reserved.
- *
- * This program and the accompanying materials are made available under the
- * terms of the Eclipse Distribution License v. 1.0, which is available at
- * http://www.eclipse.org/org/documents/edl-v10.php.
- *
- * SPDX-License-Identifier: BSD-3-Clause
- */
-
-/**
- *  This example is to show the unmarshalling of a Javabean with setters
- *  but no getters. 
- * 
- *  Author: Sekhar Vajjhala
- *
- *  $Id: Main.java,v 1.2 2009-11-11 14:17:30 pavel_bucek Exp $
- */  
-
-/**
- * OBSERVATIONS:
- *
- * 1. The Javabean USAddress contains getters but not setters. I
- *    first thought that an unmarshal call would fail. However, it
- *    succeeded. Then I remembered that JAXBRI is unmarshalling using
- *    fields. However this may not be clear to a newbie JAXB 2.0
- *    user and is therefore worth clarifying.
- *
- * 2. If AccessType.PROPERTY is the default, then the above
- *     unmarshalling would be expected to fail (probably an
- *    exception).
- *
- * 3. An attempt to unmarshal into a JAXBElement instance resulted in
- *    a ClassCastException. 
- *        Exception in thread "main" java.lang.ClassCastException: USAddress
- * 	 at Main.main(Main.java:36)
- * 
- *    Note the way I hit this problem was that I copied the
- *    unmarshal-read sample where the the unmarshal call was cast
- *    to JAXBElement instance. However, I chose to use the
- *    @XmlRootElement on USAddress but forgot to cast the return type
- *    of the Unmarshaller.unmarshal to USAddress.
- *    
```

Comment:
@lukasj Removal seems appropriate, sample doesn't work and most probably shouldn't work spec says:
```
For a JavaBean property with getter/setter methods, a setter
method is required for unmarshalling when public API (as opposed to fields) is
used i.e. either @XmlAccessorType.PUBLIC_MEMBER or
@XmlAccessorType.PROPERTY is in effect. If
@XmlAccessorType.FIELD is in effect, then unmarshalling is based on fields
and hence a setter method is not required. 
```
Default accessor type is PUBLIC_MEMBER

Reply:
@lukasj you were right about this sample, it worked because of the accessor optimization. Seems like it stopped working with this PR https://github.com/javaee/jaxb-v2/pull/1157

Reply:
https://github.com/javaee/jaxb-v2/commit/8c3171908022c681699d5d31be2a4812663fdf25